content/03.use-cases-a.md

```diff
@@ -26,12 +30,12 @@ For example, organizing a course could be broken down into: 1) developing the ma
 While there are other purpose-built platforms for this, GitHub provides a free, open-source alternative.
 
 GitHub's open nature allows colleagues to see, review and offer feedback on your process.
-Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338]; Python, with `python-ppt` [@https://python-pptx.readthedocs.io/en/latest/index.html]); and Julia, with `Remark.jl` [@https://juliapackages.com/p/remark].
+Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338], Python, with `python-ppt` (https://python-pptx.readthedocs.io/en/latest/index.html), and Julia, with `Remark.jl` (https://juliapackages.com/p/remark).
```

Comment:
```suggestion
Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338], Python, with `python-ppt` (https://python-pptx.readthedocs.io), and Julia, with `Remark.jl` (https://juliapackages.com/p/remark).
```
That redirects to https://python-pptx.readthedocs.io/en/latest/index.html

Reply:
Great thanks!